prober/derp_test.go

```diff
@@ -5,11 +5,13 @@ package prober
 
 import (
 	"context"
+	"crypto/md5"
```

Comment:
Minor nit:  Since this is a test md5 is fine, but seeing this imported is like to make the reader (and potentially static analysis tools) convince themselves of that again in the future. We might as well just use sha256 even though it's not really necessary for its properties.